### DISCUSSION TITLE
请问如何让中英文摘要之间没有自动生成的空白页？

### DISCUSSION THREAD

**Discussion by LiShuaiyi** [sjtug/SJTUThesis#989] [提问] [answered]
发现中文摘要和英文摘要之间会有自动生成的空白页，并且空白页会占一个页码，就变成中文摘要的页码是I，英文摘要是III，请问要怎么删除这个空白页呢？

**Comment by LogCreative** [comment, marked as answer]
这是因为每个摘要都单独成章，默认的 `twoside` 模式会让章从奇数页开始。

如果你准备提交电子版，在文档类上添加选项 `oneside`。
